src/pages/stacks-blockchain/technical-specs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Technical Specifications
+description: Summary of technical specifications of Stacks 2.0
+---
+
+## Consensus
+
+- Proof of Transfer (PoX) as described in [SIP-007](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-007-stacking-consensus.md)
+- Network will transition to Proof of Burn (PoB) as described in [SIP-001](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md) after 10 years. More details [here](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md).
+
+## Proof of Transfer Mining
+
+- Coinbase reward schedule:
+  - 1000 STX/block for first 4 years
+  - 500 STX/block for following 4 years
+  - 250 STX/block for subsequent 4 years
+  - 125 STX/block in perpetuity after that
+- Reward maturity window: 100 blocks, meaning leaders will earn the coinbase reward 100 blocks after the block they successfully mine.
+- Block interval: Stacks blockchain produces blocks at the same rate as the underlying burnchain. For Bitcoin, this is approximately every 10 minutes.
+
```

Comment:
if we merge this after the mining summary, we should link to the mining page here for further reading